source/postprocess/particles.cc

```diff
@@ -354,141 +354,147 @@ namespace aspect
                                                this->get_time() / year_in_seconds :
                                                this->get_time());
 
-      if (output_format=="none")
+      for (std::vector<std::string>::iterator itr_output_format = output_formats.begin();
+           itr_output_format != output_formats.end();
```

Comment:
I would prefer if you remove the `itr_` part of the name. We usually do not name variables after their type. Just call it output_format